en/getting_started/README.md

```diff
@@ -0,0 +1,96 @@
+It also provides information about optional or OS specific functionality.
+
+## Source Code
+
+Source code for *QGroundControl* is kept on GitHub here: https://github.com/mavlink/qgroundcontrol.
+It is [dual-licensed under Apache 2.0 and GPLv3](https://github.com/mavlink/qgroundcontrol/blob/master/COPYING.md).
+
+To get the source files:
+1. Clone the repo (or your fork):
+   ```
+   git clone --recursive https://github.com/mavlink/qgroundcontrol.git
+   ```
+2. Update submodules (do this each time you pull new source code):
+   ```
+   git submodule update
+   ```
+
+> **Tip** Github source-code zip files cannot be used because these do not contain the appropriate submodule source code. You must use git!
+
+
+## Build QGroundControl
+
+### Native Builds
+
+*QGroundControl* builds are supported for OSX, Linux, Windows, iOS and Android. *QGroundControl* uses [Qt](http://www.qt.io) as its cross-platform support library and uses [QtCreator](http://doc.qt.io/qtcreator/index.html) as its default build environment.
+
+- **OSX:** OSX 10.7 or higher, 64 bit, clang compiler
+  > **Warning** XCode 8 requires a workaround described below.
+- **Ubuntu:** 64 bit, gcc compiler
+- **Windows:** Vista or higher, 32 bit, [Visual Studio 2015 compiler](http://www.visualstudio.com/downloads/download-visual-studio-vs#d-express-windows-desktop)
+- **iOS:** 8.0 and higher
+- **Android:** Jelly Bean (4.1) and higher. Standard QGC is built against ndk version 19.
+- <span id="qt_version"></span>**Qt version:** 5.9.3 **only**
+
+#### Install Qt
+
+You **need to install Qt as described below** instead of using pre-built packages from say, a Linux distribution, because *QGroundControl* needs access to private Qt headers.
+1. Download the [Qt installer](http://www.qt.io/download-open-source) for the [supported Qt version)[#qt_version]. You will also need to install the *Qt Speech* package.
+   - **Ubuntu:** Set the downloaded file to executable using:`chmod +x`. Install to default location for use with **./qgroundcontrol-start.sh.** If you install Qt to a non-default location you will need to modify **qgroundcontrol-start.sh** in order to run downloaded builds.
+   - **Windows:** Make sure to install *VS 2015 32 bit* package.
+
+1. Install Additional Packages (Platform Specific)
+   - **Ubuntu:** `sudo apt-get install speech-dispatcher libudev-dev libsdl2-dev`
+   - **Fedora:** `sudo dnf install speech-dispatcher SDL2-devel SDL2 systemd-devel`
+   - **Arch Linux:** `pacman -Sy speech-dispatcher`
+   - **Windows:** [USB Driver](http://www.pixhawk.org/firmware/downloads) to connect to Pixhawk/PX4Flow/3DR Radio
+   - **Android:** [Qt Android Setup](http://doc.qt.io/qt-5/androidgs.html)
+
+#### Building using Qt Creator
+
+1. Launch *Qt Creator* and open the **qgroundcontrol.pro** project.
+1. Select the appropriate kit for your needs:
+  - **OSX:** Desktop Qt 5.9.3 clang 64 bit
```

Comment:
[Above](https://github.com/mavlink/qgc-dev-guide/pull/36/files#diff-113127c1f416c66e653663790de312a9R36) I have marked the version of QT require. It might be wort changing these to be just a link, so we only have to maintain the version in one place - e.g. 

- **Ubuntu:** Desktop Qt *[Version](https://github.com/mavlink/qgc-dev-guide/pull/36/files#diff-113127c1f416c66e653663790de312a9R36)* GCC bit

Thoughts?

Reply:
No, the link I am proposing is within this same document (up the top). The idea is that we only have to specify the text for the Qt version once in this document, and for the other places you can click the link to find the current version.

It is also possible to declare a variable for the version text in gitbook (book.json) and import this everywhere it is used. Only caveat is that the variable does not render as the imported string in Github.

Thoughts?

Reply:
Ah, I see what you are doing. I think that's ok.

Reply:
OK, so what I have actually done is put this in the book.json. In the first instance I have a comment so that anyone wanting to update this can see where they need to go.